<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sn3DDialogBack</class>
 <widget class="QDialog" name="Sn3DDialogBack">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sn3DDialogBack</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#widget_back{

border-image: url(:/images/dialog.png) 70;
border-width:70
}

QPushButton
{
color: rgb(255, 255, 255);

border-image: url(:/images/btn_normal_normal.png) 2;
border-width: 2;
}

QPushButton:hover
{
border-image: url(:/images/btn_normal_hover.png) 2;
border-width: 2; 
}

QPushButton:pressed
{
border-image: url(:/images/btn_normal_press.png) 2;
border-width: 2;
}

QPushButton:disabled
{
color: rgb(102, 102, 102);
border-image: url(:/images/btn_normal_disabled.png) 2;
border-width: 2;
}
QRadioButton {    
	color: #9b9b9b;	
    
	font: 14pt &quot;Arial&quot;;
}

QRadioButton::indicator:unchecked {
    
	color: #ffffff;
	border-image: url(:/images/ridio_norml.png);
}
QRadioButton::indicator:checked {
   color: #ffffff;
	border-image: url(:/images/rido_checked.png);
}

QLineEdit {
     border-width: 1px;
     border-style: solid;
     border-color: #606060;    
	color: rgb(231, 231, 231);

 }

QLineEdit:disabled
{

	 border-width: 1px;
     border-style: solid;
     border-color: #606060;    
	color: rgb(102, 102, 102);

}
/*QLabel{
	
	color: rgb(231, 231, 231);
}*/</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_back" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="widget_top">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Title</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_content" native="true">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>332</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>332</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
